### PR TITLE
test(cce/addon): update test cases of resource cce addon

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

update test cases of resource cce addon

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes NONE

**Special notes for your reviewer**:

**Release note**:

```release-note
update test cases of resource cce addon
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCCEAddonV3_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCCEAddonV3_basic -timeout 360m -parallel 4
=== RUN   TestAccCCEAddonV3_basic
=== PAUSE TestAccCCEAddonV3_basic
=== CONT  TestAccCCEAddonV3_basic
--- PASS: TestAccCCEAddonV3_basic (910.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       910.895s

make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCCEAddonV3_values'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCCEAddonV3_values -timeout 360m -parallel 4
=== RUN   TestAccCCEAddonV3_values
=== PAUSE TestAccCCEAddonV3_values
=== CONT  TestAccCCEAddonV3_values
--- PASS: TestAccCCEAddonV3_values (801.90s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       801.995s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_cce_addon_v3_test.go [hunk 1]
```diff
@@ -194,16 +194,40 @@ func testAccCCEAddonV3_values(rName string) string {
...
+  os                 = "EulerOS 2.5"
+  flavor_id          = "s6.large.2"
+  initial_node_count = 4
+  availability_zone  = data.huaweicloud_availability_zones.test.names[0]
+  key_pair           = huaweicloud_compute_keypair.test.name
+  scall_enable       = true
+  min_node_count     = 2
+  max_node_count     = 10
+  priority           = 1
+  type 				 = "vm"
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_addon_v3_test.go]: format error

File: huaweicloud/resource_huaweicloud_cce_addon_v3_test.go [hunk 2]
```diff
@@ -194,16 +194,40 @@ func testAccCCEAddonV3_values(rName string) string {
 	return fmt.Sprintf(`
 %s
 
+resource "huaweicloud_cce_node_pool" "test" {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_addon_v3_test.go:197]: is `huaweicloud_cce_node_pool` necessary for this testing case?

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_addon_v3_test.go:197]: yes, addon autoscaler should be used with node pool.
Scaling is not support in default node pool, so installing autoscaler without node pool is actually meaningless.

File: huaweicloud/resource_huaweicloud_cce_addon_v3_test.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_addon_v3_test.go]: fixed

**Pull request merged**